src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -77,7 +77,7 @@ public void execute_storageThrowsIoException_throwsCommandException() {
         logic = new LogicManager(model, storage);
 
         // Execute add command
-        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY
+        String addCommand = TagCommand.COMMAND_WORD + NAME_DESC_AMY
                 + FILE_ADDRESS_DESC_AMY;
```

Comment:
Seems that this test also need to be changed in the future. But I think maybe this does not matter now.

Reply:
We need to change all the entities in testUtil.